### PR TITLE
Disable upstart integration on artful+.

### PR DESCRIPTION
**Pull request by xnox**

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -121,6 +121,16 @@ override_dh_install:
 	dh_install
 
 override_dh_installinit:
+ifeq (,$(findstring $(DEB_DISTRIBUTION), trusty utopic vivid wily xenial yakkety zesty))
```
Comment (stgraber) [inline debian/rules]: Can be reduced to just "trusty xenial zesty".

**Comment by hallyn** [comment]
What is the advantage of doing this?

**Comment by stgraber** [comment]
That's a good question. @xnox

**Comment by xnox** [comment]
@hallyn @stgraber upstart is no longer shipped by artful thus these files are dead weight. they take up space in debs, images, dpkg conffile database and are bloat which results in more things stated in maintainer scripts or whenever anybody does `grep -r /etc`. In artful lxd/cloud-image there are now only three packages remaining that ship maintainer scripts lxd, cloud-init and openssh. This is patch for lxd, openssh patch submitted to BTS and cloud-init I am working on. If these three packages are fixed there would be no `/etc/init` in default artful lxd container / images.

File: debian/rules (hunk 1 above)
Comment (stgraber) [inline debian/rules]: @xnox did you see this comment?

Comment (xnox) [inline debian/rules]: @stgraber i have not seen this comment no. "trusty xenial zesty" sounds good to me. Will fix.

**Comment by xnox** [comment]
@stgraber updated list of suites to enable upstart integration for; and updated changelog version number + maintscript version number to match.

**Comment by stgraber** [comment]
Changelog entry should be UNRELEASED or just not there at all as we won't be uploading just for this one change.

**Comment by xnox** [comment]
@stgraber well, the versions must match between .maintscripts and changelog =/ and I hope this will be uploaded into artful before artful release.

**Pull request merged**